search-service/src/main/kotlin/db/migration/V0_28__JsonLd_migration.kt

```diff
@@ -208,7 +208,7 @@ class V0_28__JsonLd_migration : BaseJavaMigration() {
         attributePayload: Map<String, List<Any>>,
         ngsiLdAttributeInstance: NgsiLdAttributeInstance
     ) {
-        val createdAt = ngsiLdAttributeInstance.createdAt
+        val createdAt = ngsiLdAttributeInstance.createdAt ?: ZonedDateTime.parse("1970-01-01T00:00:00Z")
```

Comment:
why not doing in the same way than it is done in `createTeaEntry()`?

Reply:
I do this